backend/trade/views.py

```diff
@@ -104,6 +154,17 @@ def reject(self, request, pk=None):
 
         trade.update_trade_status()
         trade.save()
+
+        system_chat = get_object_or_404(TradeChat, trade_id=trade.id)
+        ChatMessage.objects.create(
+            trade_chat=system_chat,
+            sender_type=ChatMessage.SYSTEM,
+            event_type=trade.trade_status,
+            message=f"On {datetime.now().strftime('%Y-%m-%d %H:%M:%S')}, "
```

Comment:
Let's remove "On <date>... 

It won't be necessary because the date will appear on the message itself. Remove it on other occurrences too.

Reply:
And let's use past simple tense - User1 accepted rejected the trade...

Reply:
Done.